Buildings/Fluid/Movers/Data/Fans/EnglanderNorford1992/Return.mo

```diff
@@ -0,0 +1,23 @@
+within Buildings.Fluid.Movers.Data.Fans.EnglanderNorford1992;
+record Return "Data for the return fan used in Englander and Norford (1992)"
+  extends Generic(
+    etaHydMet=Buildings.Fluid.Movers.BaseClasses.Types.EfficiencyMethod.PowerCurve,
+    speed_rpm_nominal=800,
+    power(V_flow={10.2875112309074,24.9775381850853,35.4447439353099,53.414195867026},
+          P={29828,44742,51050.622,44742}),
+    pressure(V_flow={10.2875112309074,24.9775381850853,35.4447439353099,53.414195867026},
+             dp={1453.61155698234,1371.42857142857,1052.9695024077,0}));
```

Comment:
Please format the data so that they are easier to read.

Reply:
@mwetter
Do you want me to apply some format to just the two Englander data files or all of the data records?

Reply:
Formatted in f40adfd71d5db51a65db546f38607bd4c0eef3ba and 4662c8720e861e6051a4bca605383a4e491c3e33.